charts/mattermost-push-proxy/templates/configmap-apple-cert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mattermost-push-proxy.fullname" . }}-apple-cert-path
+  labels:
+    chart: {{ include "mattermost-push-proxy.chart" . }}
```

Comment:
can we use the new labels? like https://github.com/mattermost/mattermost-kubernetes/blob/master/charts/mattermost-enterprise-edition/templates/deployment-mattermost-app.yaml#L6

Reply:
@cpanato Mattermost-push-proxy adapted to use new labels. Let me know if anything is missing.